sceptre/scipool/README.md

```diff
@@ -31,15 +31,44 @@ Requirements:
 * Run `pre-commit install` to install the git hook.
 
 ## Testing
+
+### Pre-Commit
 As a pre-deployment step we syntatically validate our sceptre and
 cloudformation yaml files with [pre-commit](https://pre-commit.com).
 
 Please install pre-commit, once installed the file validations will
-automatically run on every commit.  Alternatively you can manually
+automatically run on every commit. Alternatively you can manually
 execute the validations by running `pre-commit run --all-files`.
 Please install pre-commit, once installed the file validations will
 automatically run on every commit.
 
+### Functional Testing
+The process to test the functionality of an AMI and it's integration with our
+Service Catalog products is to first create a test AMI, upload a modified
```

Comment:
Below you say to create the new product via a PR, not by uploading to S3.  Can you please be consistent?

Reply:
I'm trying to describe both the underlying process and the automation around it. I've reworded this to hopefully explain that a bit more